101-hdinsight-autoscale-schedulebased/README.md

```diff
@@ -0,0 +1,20 @@
+# HDInsight (Spark + Schedule based Autoscale)
+
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/101-hdinsight-custom-ambari-db/PublicLastTestDate.svg" />&nbsp;
```

Comment:
see pipeline results for proper button format

Reply:
Updated README and metadata.json.